static/js/main.js

```diff
@@ -423,8 +423,25 @@ function fetchVersion() {
         });
 }
 
+// Initialize hotkey functionality.
+function initializeHotkeys() {
+    document.addEventListener('keydown', (event) => {
+        if (event.code === 'KeyB') {
+            toggleSelection();
+        }
+        if (event.code === 'KeyF') {
+            captureFrame();
+        }
+        if (event.code === 'KeyS') {
+            save();
+        }
```

Comment:
There's a couple important things here.

1) NEVER have disjointed if statements next to each other without whitespace between them.
It is way to easy for a reader to accidentally see them as else-ifs.
Remember how I am always taking about vertical visual "blocks" of code.
(This case will actually not apply because of the rest of this comment, but this point is very important.)

2) NEVER use an if (verses an else-if or else) for non-disjoint cases.
Although the code will work just fine, you want to indicate to the readers that these cases cannot both happen at the same time.

Note that both of these are all about making it easy for the reader and making it hard for the reader to misread your code.

Reply:
Do you mean:
```
if {
   xxx 
} else if {
    yyy 
} else {
   zzz 
}
```

If so yes, we call that "cuddling".
Different style may not cuddle, but these two points are unrelated to cuddling (since that is the same conditional (just different branches)).

If not, then I don't understand.
I guess the answer would be "no", since it doesn't make sense as written.